images/releng/k8s-ci-builder/variants.yaml

```diff
@@ -2,28 +2,28 @@ variants:
   default:
     CONFIG: default
     GO_VERSION: '1.21.6'
-    OS_CODENAME: 'bullseye'
+    OS_CODENAME: 'bookworm'
   next:
     CONFIG: next
     GO_VERSION: '1.22rc1'
     OS_CODENAME: 'bookworm'
   '1.30':
     CONFIG: '1.30'
     GO_VERSION: '1.22rc1'
-    OS_CODENAME: 'bullseye'
+    OS_CODENAME: 'bookworm'
   '1.29':
     CONFIG: '1.29'
     GO_VERSION: '1.21.6'
-    OS_CODENAME: 'bullseye'
+    OS_CODENAME: 'bookworm'
   '1.28':
     CONFIG: '1.28'
     GO_VERSION: '1.20.13'
-    OS_CODENAME: 'bullseye'
+    OS_CODENAME: 'bookworm'
```

Comment:
Change this as https://testgrid.k8s.io/sig-release-1.28-blocking#build-1.28 started to fail as well.